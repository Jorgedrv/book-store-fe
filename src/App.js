import React from 'react';

const App = () => {
    return <div>My first app, let's staterd</div>;
};

export default App;