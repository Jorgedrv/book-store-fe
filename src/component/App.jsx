import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import Layout from './layout/Layout';
import BookContainer from './book/BookContainer';
import Login from './user/Login';
import Register from './user/Register';
import Auth from './auth/index';
import { SnackbarProvider } from 'notistack';

const App = () => {
    return (
        <SnackbarProvider maxSnack={3}>
            <Router>
                <Layout>
                    <Switch>
                        <Route path='/login'>
                            <Login />
                        </Route>
                        <Route path='/register'>
                            <Register />
                        </Route>
                        <Route exact path='/'>
                            <Auth>
                                <BookContainer />
                            </Auth>
                        </Route>
                    </Switch>
                </Layout>
            </Router>
        </SnackbarProvider>
    );
};

export default App;