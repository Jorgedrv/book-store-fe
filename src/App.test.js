import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App component', () => {
    it('App test should render with error', () => {
        const { asFragment } = render(<App/>);
        expect(asFragment()).toMatchSnapshot();
    });
});