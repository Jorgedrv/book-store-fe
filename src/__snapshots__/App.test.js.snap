// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component App test should render with error 1`] = `
<DocumentFragment>
  <div>
    My first app, let's staterd
  </div>
</DocumentFragment>
`;
